cfme/physical/overview.py

```diff
@@ -0,0 +1,32 @@
+from navmazing import NavigateToAttribute
+from widgetastic_manageiq import StatusBox
```

Comment:
Required:
Please sort imports as per libs
1. python core
2. third party
3. local

Reply:
- no core imports
- navmazing and widgetstatic
- local imports
aren't they correct?

Reply:
`widgetastic_manageiq` is part of the integration test. Ya, we are planning to move it but for now teat it as a local import.